samples/model-builder/model_registry/delete_model_version_sample.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_model_registry_delete_model_version_sample]
+
+from google.cloud import aiplatform
+
+
+def delete_model_version_sample(
+    model_id: str, version: str, project: str, location: str
+):
+    """
+    Delete a Model.
```

Comment:
Model version? 

also, sometimes you use Model and sometimes model, we should try to be consistent.

Reply:
Good catch! About the use of Model and model it depends if I am referring to a Model Resource (Model) or its instantiation (model). I will make Model resource more explicit to avoid confusion